bin/database.py

```diff
@@ -1112,14 +1113,31 @@ def upgrade_to_30():
     process_cursor(cursor, upgrade_to_30_closure_coll, context = 'projects')
 
 
-def upgrade_schema():
+def upgrade_schema(force_from = None):
     """
     Upgrades db to the current schema version
 
     Returns (0) if upgrade is successful
     """
 
     db_version = get_db_version()
+
+    if isinstance(db_version,int) and db_version >= force_from:
+        if(force_from):
+            db_version = force_from
```

Comment:
I would switch these if statements around (1125 and 1126). `int >= None` functionally works, but the conditional makes more sense when written as "if you are attempting to use a force from, make sure the db_version is greater than or equal to the force_from".